meta-resin-common/recipes-devtools/kernel-modules-headers/kernel-modules-headers.bb

```diff
@@ -7,7 +7,10 @@ SECTION = "devel/kernel"
 LICENSE = "Apache-2.0"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=89aea4e17d99a7cacdbeed46a0096b10"
 
-DEPENDS = "virtual/kernel"
+DEPENDS = " \
+    virtual/kernel \
+    bc-native \
```

Comment:
Is this script using bc? I don't recall it

Reply:
The kernel build scripts require bc for timeconst.sh.